src/style.scss

```diff
@@ -141,211 +147,50 @@ slot {
-        border: 1px solid var(--ds-color-ui-default-default, $ds-color-ui-default-default);
-      }
-    }
-
-    &:active {
-      background-color: var(--ds-color-ui-bkg-hover-default, $ds-color-ui-bkg-hover-default);
-      border: 1px solid var(--ds-color-ui-default-default, $ds-color-ui-default-default);
-    }
-
+  &[variant='secondary'] {
     &:disabled {
       cursor: not-allowed;
       transform: unset;
-
-      color: rgba($ds-color-text-link-default, $disabledOPacity);
-      background-color: rgba($ds-color-background-lightest, $disabledOPacity);
-      border: 1px solid rgba($ds-color-ui-default-default, $disabledOPacity);
-
-      @media (hover: hover) {
-        &:hover {
-          color: rgba($ds-color-text-link-default, $disabledOPacity);
-          background-color: rgba($ds-color-background-lightest, $disabledOPacity);
-          border: 1px solid rgba($ds-color-ui-default-default, $disabledOPacity);
-        }
-      }
     }
   }
 
-  // auro-button--tertiary
-  &[variant='tertiary'],
-  &--tertiary {
-    background-color: $auro-rgb-color-base-black-03;
-    border-color: transparent;
-    color: var(--ds-color-text-link-default, $ds-color-text-link-default);
-
-    @media (hover: hover) {
-      &:active,
-      &:hover {
-        background-image: linear-gradient(var(--ds-color-ui-bkg-hover-default), var(--ds-color-ui-bkg-hover-default));
-        border-color: transparent;
-      }
-    }
-
-    &:active {
-      background-color: var(--ds-color-ui-bkg-hover-default, $ds-color-ui-bkg-hover-default);
-      border-color: transparent;
-    }
-
+  &[variant='tertiary'] {
     &:disabled {
       cursor: not-allowed;
       transform: unset;
-
-      color: rgba($ds-color-text-link-default, $disabledOPacity);
-      background-color: $auro-rgb-color-base-black-03;
-      border: 1px solid transparent;
-
-      @media (hover: hover) {
-        &:hover {
-          background-color: $auro-rgb-color-base-black-03;
-          border: 1px solid transparent;
-        }
-      }
     }
   }
 
-  &Ondark {
-    background-color: var(--ds-color-ui-default-inverse, $ds-color-ui-default-inverse);
-    border: 1px solid var(--ds-color-ui-default-inverse, $ds-color-ui-default-inverse);
-    color: var(--ds-color-text-primary-default);
-
-    // auro-buttonOndark hover
-    @media (hover: hover) {
-      &:active,
-      &:hover {
-        background-image: linear-gradient(var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse), var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse));
-        border: 1px solid var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse);
-      }
-    }
-
-    &:active {
-      background-color: var(--ds-color-ui-active-inverse, $ds-color-ui-active-inverse);
-      border: 1px solid var(--ds-color-ui-active-inverse, $ds-color-ui-active-inverse);
-    }
-
+  &[ondark] {
```

Comment:
I know this has nothing to do with your code, but if we support `ondark` I feel like it should also be a `variant`